.github/workflows/sanity-check.yml

```diff
@@ -16,13 +16,10 @@ jobs:
         env:
           pull_request_number: ${{ github.event.pull_request.number }}
         run: |
-          curl --silent -X "GET" https://api.github.com/repos/debezium/debezium/pulls/$pull_request_number/commits | jq '.[] | {message: .commit.message}' | jq -r '.message' >> COMMIT_MESSAGE.txt
-          while IFS=" " read -r MSG;
-          do
-            if [[ ! $MSG =~ (^DBZ-[[:digit:]]+)|(\[release\])|(\[jenkins-jobs\])|(\[docs\]) ]]; then
-              echo "::set-output name=PREFIX::false"
-            fi
-          done < COMMIT_MESSAGE.txt
+          COMMIT_MSG=$(curl --silent -X "GET" https://api.github.com/repos/debezium/debezium/pulls/$pull_request_number/commits | jq '.[] | {message: .commit.message}' | jq -r '.message' | head -n1)
+          if [[ ! "$COMMIT_MSG" =~ (^DBZ-[[:digit:]]+)|(\[release\])|(\[jenkins-jobs\])|(\[docs\]) ]]; then
+            echo "::set-output name=PREFIX::false"
+          fi
```

Comment:
@ani-sha can we talk about this as it fixes the wrong bot-comment? [#2691]

Reply:
@rk3rn3r, this would only work where we have a single commit. But in the case of multiple commits, it would only be checking the first one and passing the CI.

Reply:
@rk3rn3r How about comparing the total number of commits in a pull request with the number of matching lines that start with the prefix; if the count doesn't match create a comment. 
This might not be an ideal approach but do you have anything else in mind for handling the multi-line commit messages?